src/main/java/com/parasoft/demoapp/dto/InventoryOperation.java

```diff
@@ -0,0 +1,5 @@
+package com.parasoft.demoapp.dto;
+
+public enum InventoryOperation {
+    INCREASE_FOR_ORDER_DENIAL, DECREASE_FOR_ORDER_CREATION, INCREASE_FOR_ORDER_CREATION_FAILED
```

Comment:
@rwang-parasoft Besides different error message, is there any other reason why we need to differentiate "INCREASE_FOR_ORDER_DENIAL" and "INCREASE_FOR_ORDER_CREATION_FAILED"?